frontend/src/components/fundingAgreements/ManageFundingAgreements.vue

```diff
@@ -78,25 +101,25 @@ export default {
   computed: {
     ...mapState(useOrganizationStore, ['organizationId']),
     filteredFundingAgreements() {
-      return this.fundingAgreements.filter(
-        (item) =>
-          (this.showFundingAgreements && !item.isModification) || (this.showModifications && item.isModification),
-      );
+      const agreements = isEmpty(this.selectedTerms)
+        ? this.fundingAgreements
+        : this.fundingAgreements.filter((agreement) => this.selectedTerms.includes(agreement.fundingAgreementTerm));
+      return this.sortFundingAgreements(agreements);
```

Comment:
I don't think we want to sort on filter. Normally the default sort applies only when loading the page.